config/config.exs

```diff
@@ -20,11 +20,26 @@ config :signs_ui,
 # HTTP config
 config :signs_ui, :redirect_http?, false
 
+config :signs_ui, SignsUiWeb.AuthManager,
+  issuer: "signs_ui",
+  secret_key: "test key"
+
 # Configures Elixir's Logger
 config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:user_id]
 
+config :ueberauth, Ueberauth,
+  providers: [
+    cognito: {Ueberauth.Strategy.Cognito, []}
+  ]
+
+config :ueberauth, Ueberauth.Strategy.Cognito,
```

Comment:
Maybe have this in `config/prod.exs` and a different one for local development?

Reply:
Good idea. I created and configured an Ueberauth strategy for use in the dev and test environments that just logs you in, rather than redirecting away to AWS.